modules/ROOT/pages/create-application.adoc

```diff
@@ -5,7 +5,7 @@
 
 The last step of this Getting Started tutorial is to create an application.
 
-To create and deploy a first application in your local environment, use the *xref:applicationCreation.adoc[Application descriptor]*. This editor defines the skeleton of an application, as it contains references to resources installed in Bonita Portal and used by a given application. It will also help deploy the application directly from Bonita Studio.
+To create and deploy a first application in your local environment, use the *xref:applicationCreation.adoc[Application descriptor]*. This editor defines the skeleton of an application, as it contains references to resources installed in the Bonita Runtime and used by a given application. It will also help deploy the application directly from Bonita Studio.
```

Comment:
To create and deploy a first application in your local environment, => To create and deploy an application in your local environment,

If the deploiement process is the same for any application.

Reply:
Added